baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/DangerousThrowableMessageSafeArg.java

```diff
@@ -63,8 +61,9 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
         ExpressionTree safeValueArgument = args.get(1);
         if (THROWABLE_MESSAGE_METHOD.matches(safeValueArgument, state)) {
             return buildDescription(tree)
-                    .setMessage("Do not use throwable messages as SafeArg values. "
-                            + "SafeLoggable.getLogMessage is guaranteed to be safe.")
+                    .setMessage(
+                            "Do not use throwable messages as SafeArg values. "
+                                    + "SafeLoggable.getLogMessage is guaranteed to be safe.")
```

Comment:
wonder why it didn't inline the string after `.setMessage(`.
I think we need to set a Breakability of `BREAK_HERE` for binary operators to make this work